scripts/deploy.py

```diff
@@ -374,10 +370,14 @@ def check_travis_and_circleci_tests(current_branch_name):
     circleci_url = 'https://circleci.com/gh/%s/workflows/oppia' % (
         github_username)
 
-    if requests.get(travis_url).status_code != 200:
+    try:
+        python_utils.url_open(travis_url)
+    except Exception:
         travis_url = 'https://travis-ci.org/oppia/oppia/branches'
 
-    if requests.get(circleci_url).status_code != 200:
+    try:
+        python_utils.url_open(circleci_url)
+    except Exception:
```

Comment:
Not sure - since the [function](https://github.com/oppia/oppia/blob/develop/python_utils.py#L295) itself uses `urllib2` or `urllib.request` based on python version, so it will be harder to catch exceptions. Also, I think it is ok to redirect the user to Oppia travis/circleci urls in case of any exception since that will definitely exist.